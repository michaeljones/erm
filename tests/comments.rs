extern crate erm;
extern crate im;
extern crate insta;
extern crate logos;
extern crate unindent;

mod common;

mod comments {

    use common::eval;

    #[test]
    fn single_line_comment() {
        let src = r#"
        module Main exposing (..)

        -- This is a comment
        main : List String -> String
        main args =
            "Hello comments"
        "#;
        let result = eval(src, None);
        insta::assert_snapshot!(result);
    }
}
