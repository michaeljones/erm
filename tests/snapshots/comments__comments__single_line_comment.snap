---
source: tests/comments.rs
expression: result

---
Hello comments
